### ISSUE TITLE
nvidia: The SAMPLES values of the read and draw framebuffers should match.

### ISSUE THREAD

**Issue by CubesAndCubes** [ValveResourceFormat/ValveResourceFormat#723]
### Describe the bug

I cannot view CS2 maps, the render is completely black.

![img](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/120466698/e82a0cd8-0e5a-4920-8c1b-1f835f057ad7)

### Relevant log output or exception with stacktrace

```shell
[19:52:55.385] [OpenGL] [High Api Error] GL_INVALID_OPERATION error generated. The SAMPLES values of the read and draw framebuffers should match.
```

### What game does this happen in

CS2

### Reference file in a Source 2 game

`de_inferno.vmap` in `...\Counter-Strike Global Offensive\game\csgo\maps\de_inferno.vpk`

### Expected behavior

I can view CS2 maps.

### What version of VRF are you using? On what platform?

v8.1.3093+cdfbda66d9204bbab7d2d98c60f1fb8a336c660f, Microsoft Windows NT 10.0.22631.0

**Comment by CubesAndCubes** [comment]
My GPU:

```
[OpenGL] GPU: NVIDIA GeForce RTX 4090/PCIe/SSE2, Driver: 4.6.0 NVIDIA 551.23
```

**Comment by xPaw** [comment]
Decompilation is not a bug, you just didn't put it into the correct folder: https://github.com/ValveResourceFormat/ValveResourceFormat/discussions/619

**Comment by xPaw** [comment]
Did you modify any of the settings in nvidia panel? Like anything related to anti aliasing or multisampling.

**Comment by CubesAndCubes** [comment]
I cannot remember that I did, but resetting the 3D settings did indeed fix the issue. However, I'm now unable to reproduce a configuration that causes the renderer to bug out again. Perhaps my configuration file was somehow corrupted.

**Comment by FifthGoblin** [comment]
> I cannot remember that I did, but resetting the 3D settings did indeed fix the issue. However, I'm now unable to reproduce a configuration that causes the renderer to bug out again. Perhaps my configuration file was somehow corrupted.

which 3D settings did you reset? Could you explain a bit further? I am having the same issue. 

Thanks a bunch.

**Comment by CubesAndCubes** [comment]
> > I cannot remember that I did, but resetting the 3D settings did indeed fix the issue. However, I'm now unable to reproduce a configuration that causes the renderer to bug out again. Perhaps my configuration file was somehow corrupted.
> 
> which 3D settings did you reset? Could you explain a bit further? I am having the same issue.
> 
> Thanks a bunch.

The 3D settings within the Nvidia Control Panel

**Comment by schwartled** [comment]
I got it,I reproduced a configuration that cause this bug.If you choose the High quality NVD processor  option in Preferred graphics processor in the 3D settings within the Nvidia Control Panel could cause this bug，but if you choose the Auto option,it won't.

**Comment by kristiker** [comment]
Potentially solved by the recently merged #790
